README.md

```diff
@@ -1,70 +1,188 @@
 
-### Advanced Configuration
+### Setup
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+- Clone this [repository](git@github.com:bhushan354/Full-Stack-Capstone-Frontend.git) to your desired folder:
+
+- Run this command in your command line interface:
+
+```sh
+  cd [YOUR FOLDER]
+  git git@github.com:bhushan354/Full-Stack-Capstone-Frontend.git
+  cd hello-rails-front-end
+ 
+```
+
+- Update the database.yml with your database credentials under the development section
+
+
+### Usage
+
+- To run the project, execute the following command:
+
+```sh
+   npm install --force
+   npm start
+```
 
 ### Deployment
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+**This project is deployed by the author, no permission for deployment by any other client.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Umauir Muqeem**
+
+- GitHub: [@bhushan354](https://github.com/bhushan354)
+- Twitter: [@umair_ra0](https://twitter.com/umair_ra0)
+- LinkedIn: [Umair Muqeem](https://linkedin.com/in/engr-umair-muqeem)
+
+👤 **Eugene Omwomo**
+
+- GitHub: [@Omwomo](https://github.com/Omwomo)
+- Twitter: [@eugeneoduor9](https://twitter.com/eugeneoduor9)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/Omwomo)
+
+👤 **Bhushan Deshmukh**
+
+- GitHub: [@bhushan354](https://github.com/bhushan354)
+- LinkedIn: [Bhushan Deshmukh](https://www.linkedin.com/in/bhushan-deshmukh-5777851b1/)
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- **Add info about each category**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/bhushan354/Full-Stack-Capstone-Frontend/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, kindly drop a start ⭐️ for the [repository](git@github.com:bhushan354Full-Stack-Capstone-Frontend.git);
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to express my heartfelt gratitude to **Microvere** for the invaluable learning experience they have provided. Also I would like to thank to Stanley osagie for his support during react.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ]  On the "License" section of your readme, the link is broken because it points to a file that does not exist in your project.

Kindly fix the link so it points to the correct file `MIT.md`.

That way, your app will be well documented 💪